### PR TITLE
Back persistence

### PR DESCRIPTION
**Pull request by loicbefve**
Branche avec la DB postgresql

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,3 +47,27 @@ In `commando-covid` directory:
 
 `make -C server start` to run the API
 `make start` to run the frontend
+
+
+## Base de données:
+
+Les modèles de bases de données sont situès dans le répertoire server/db_model/models
```
Comment (Kmaschta) [inline README.md:54]:
```suggestion
Les modèles de bases de données sont situés dans le répertoire server/db_model/models
```

File: README.md [hunk 2]
```diff
@@ -26,3 +47,27 @@ In `commando-covid` directory:
...
+
+On utilise alembic https://alembic.sqlalchemy.org/en/latest/autogenerate.html pour les migrations/évolutions de la base de données  
+
+Pour appliquer une mise à jour du schéma de la base de données il faut donc faire:
+
+```bash
+python server d_manege.py db migrate
```
Comment (Kmaschta) [inline README.md:61]:
```suggestion
python server db_manage.py db migrate
```

File: README.md [hunk 3]
```diff
@@ -26,3 +47,27 @@ In `commando-covid` directory:
...
+python server d_manege.py db upgrade
```
Comment (Kmaschta) [inline README.md:62]:
```suggestion
python server db_manage.py db upgrade
```

File: server/config.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+from dotenv import load_dotenv
+load_dotenv()
+
+class Config(object):
+    DEBUG = False
+    TESTING = False
+    CSRF_ENABLED = True
+    SECRET_KEY = 'this-really-needs-to-be-changed'
```
Comment (Kmaschta) [inline server/config.py:10]: Sure :p

File: server/config.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+
+
+
+class ProductionConfig(Config):
+    DEBUG = False
+
+
+class StagingConfig(Config):
+    DEVELOPMENT = True
```
Comment (Kmaschta) [inline server/config.py:32]: Mind you just use the environment variables for that kind or configuration?
IMHO, an application must not be aware of its environment, just to its configuration

File: server/migrations/versions/909b464cd79b_.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+# revision identifiers, used by Alembic.
+revision = '909b464cd79b'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('users',
```
Comment (Kmaschta) [inline server/migrations/versions/909b464cd79b_.py:21]: We would have an SSO service developed by our friends at Nancyclotep, so users are already stored on their end
Hence, we just need to reference the data with a user identifier

**Review by Kmaschta — CHANGES_REQUESTED** [review]
Nice tho! I've always wanted to try alembic

File: server/migrations/versions/909b464cd79b_.py (hunk 6 above)
Comment (loicbefve) [reply server/migrations/versions/909b464cd79b_.py:21]: Great I'll remove this table

**Pull request closed without merging**